### PR TITLE
[Crop] output framerate

### PR DESCRIPTION
**Pull request by jaeyun-jung**
While caps negotiation, set output framerate from input configuration.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_crop/tensor_crop.c [hunk 1]
```diff
@@ -400,10 +403,28 @@ gst_tensor_crop_negotiate (GstTensorCrop * self)
...
+
+    walk = self->collect->data;
+    while (walk) {
+      cpad = (GstTensorCropPadData *) walk->data;
+
+      if (config.rate_n < 0 ||
+          gst_util_fraction_compare (cpad->config.rate_n, cpad->config.rate_d,
+              config.rate_n, config.rate_d) < 0) {
+        config.rate_n = cpad->config.rate_n;
+        config.rate_d = cpad->config.rate_d;
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_crop/tensor_crop.c:421]: break?

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_crop/tensor_crop.c:421]: No, to find min rate from all pads.

Comment (dongju-chae) [reply gst/nnstreamer/tensor_crop/tensor_crop.c:421]: ah, my bad.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by helloahn — APPROVED** [review]
LGTM

**Pull request merged**